features/webshop.feature

```diff
@@ -0,0 +1,16 @@
+Feature: User can access webshop page
+  As a user
+  In order to see products
+  I would like to be able to visit the shop page
+
+  Background:
+    Given the following product exist
+      | title   | description   | price |
+      | T-shirt | Blue t-shirt  | 50    |
+      | Mug     | Nice mug      | 20    |
+    When I visit the "Landing" page
```

Comment:
Since you only have a single scenario for now, I'd say this step and the next one should be part of the scenario and not the background step

Reply:
Ok